### PR TITLE
Bce 13148 left panel file tree

### PR DESCRIPTION
**Pull request by saarschweid**

### CODE REVIEW

File: src/main/kotlin/com/bridgecrew/results/BaseCheckovResult.kt [hunk 1]
```diff
@@ -1,5 +1,7 @@
 package com.bridgecrew.results
 
+import java.nio.file.Path
```
Comment (SteveVaknin) [inline src/main/kotlin/com/bridgecrew/results/BaseCheckovResult.kt:3]: why both this and `kotlin.io.path.Path`?

File: src/main/kotlin/com/bridgecrew/ui/vulnerabilitiesTree/CheckovFileTreeNode.kt [hunk 2]
```diff
@@ -0,0 +1,37 @@
+package com.bridgecrew.ui.vulnerabilitiesTree
+
+import com.intellij.icons.AllIcons
```
Comment (SteveVaknin) [inline src/main/kotlin/com/bridgecrew/ui/vulnerabilitiesTree/CheckovFileTreeNode.kt:3]: from my web experience, importing a lot of icons is intensive and the best practice is to import only what you use

File: src/main/kotlin/com/bridgecrew/ui/vulnerabilitiesTree/CheckovFileTreeNode.kt [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import icons.CheckovIcons
```
Comment (SteveVaknin) [inline src/main/kotlin/com/bridgecrew/ui/vulnerabilitiesTree/CheckovFileTreeNode.kt:4]: perhaps create a module of icons and export from there both the default and the custom ones you use?

**Review by SteveVaknin — APPROVED** [review]

File: src/main/kotlin/com/bridgecrew/ui/vulnerabilitiesTree/CheckovFileTreeNode.kt (hunk 2 above)
Comment (saarschweid) [reply src/main/kotlin/com/bridgecrew/ui/vulnerabilitiesTree/CheckovFileTreeNode.kt:3]: We're not actually importing any icon until we're using it, so there's no benefit in importing them explicitly. For example, the `AllIcons.Nodes.Folder` under the hood calls `load("nodes/folder.svg", 7796752220391063365L, 0);`

File: src/main/kotlin/com/bridgecrew/ui/vulnerabilitiesTree/CheckovFileTreeNode.kt (hunk 3 above)
Comment (saarschweid) [reply src/main/kotlin/com/bridgecrew/ui/vulnerabilitiesTree/CheckovFileTreeNode.kt:4]: I don't see the benefit, I actually like the idea of using project based and platform based icons 😄

**Pull request merged**